com.unity.cinemachine/Runtime/Core/Predictor.cs

```diff
@@ -147,5 +129,161 @@ public static Vector3 Damp(Vector3 initial, float dampTime, float deltaTime)
                 initial[i] = Damp(initial[i], dampTime, deltaTime);
             return initial;
         }
+
+        /// <summary>Get a damped version of a quantity.  This is the portion of the
+        /// quantity that will take effect over the given time.</summary>
+        /// <param name="initial">The amount that will be damped</param>
+        /// <param name="dampTime">The rate of damping.  This is the time it would
+        /// take to reduce the original amount to a negligible percentage</param>
+        /// <param name="deltaTime">The time over which to damp</param>
+        /// <returns>The damped amount.  This will be the original amount scaled by
+        /// a value between 0 and 1.</returns>
+        // Internal for testing
+        internal static float StandardDamp(float initial, float dampTime, float deltaTime)
+        {
+            if (dampTime < Epsilon || Mathf.Abs(initial) < Epsilon)
+                return initial;
+            if (deltaTime < Epsilon)
+                return 0;
+            return initial * (1 - Mathf.Exp(kLogNegligibleResidual * deltaTime / dampTime));
+        }
+
+        /// <summary>
+        /// Get a damped version of a quantity.  This is the portion of the
+        /// quantity that will take effect over the given time.
+        /// 
+        /// This is a special implementation that attempts to increase visual stability 
+        /// in the context of an unstable framerate.
+        /// 
+        /// It relies on AverageFrameRateTracker to track the average framerate.
+        /// </summary>
+        /// <param name="initial">The amount that will be damped</param>
+        /// <param name="dampTime">The rate of damping.  This is the time it would
```

Comment:
This does not seem to be accurate.
In our UnitTest, standardDamp reaches ~99%, where as stableDamp reaches ~85%

Reply:
> This does not seem to be accurate. In our UnitTest, standardDamp reaches ~99%, where as stableDamp reaches ~85%

Because you didn't set Damper.AverageFrameRateTracker.SetDampTimeScale

Reply:
Ok. I understand now. 👍